notebooks/k_means.livemd

```diff
@@ -0,0 +1,559 @@
+# K-means tutorial
+
+```elixir
+Mix.install(
+  [
+    {:stb_image, "~> 0.5.2"},
+    {:kino, github: "livebook-dev/kino", override: true},
+    {:nx, github: "elixir-nx/nx", sparse: "nx", override: true},
+    {:exla, "~> 0.3"},
+    {:explorer, "~> 0.2.0"},
+    {:scholar, path: "/home/mateusz/Dashbit/scholar"},
```

Comment:
I was trying to run this notebook on my machine and got a problem with this path.

Since we're using path option for a mix dependency, assuming that the person running this notebook is running it within the scholar git repo, maybe we could change that hardcoded path to a relative one, like this:

```elixir
{:scholar, path: Path.join(__DIR__, "..")},
```